docs/csharp/programming-guide/concepts/linq/how-to-parse-a-string.md

```diff
@@ -4,37 +4,46 @@ ms.date: 07/20/2015
 ms.assetid: 81e5686c-9658-42d8-a7e3-b11be0a2c98b
 ---
 # How to: Parse a String (C#)
-This topic shows how to parse a string to create an XML tree in C#.  
-  
-## Example  
- The following C# code shows how to parse a string.  
-  
-```csharp  
-XElement contacts = XElement.Parse(  
-    @"<Contacts>  
-        <Contact>  
-            <Name>Patrick Hines</Name>  
-            <Phone Type=""home"">206-555-0144</Phone>  
-            <Phone type=""work"">425-555-0145</Phone>  
-            <Address>  
-            <Street1>123 Main St</Street1>  
-            <City>Mercer Island</City>  
-            <State>WA</State>  
-            <Postal>68042</Postal>  
-            </Address>  
-            <NetWorth>10</NetWorth>  
-        </Contact>  
-        <Contact>  
-            <Name>Gretchen Rivas</Name>  
-            <Phone Type=""mobile"">206-555-0163</Phone>  
-            <Address>  
-            <Street1>123 Main St</Street1>  
-            <City>Mercer Island</City>  
-            <State>WA</State>  
-            <Postal>68042</Postal>  
-            </Address>  
-            <NetWorth>11</NetWorth>  
-        </Contact>  
-    </Contacts>");  
-Console.WriteLine(contacts);  
-```  
+
+This topic shows how to parse a string to create an XML tree in C#.
+
+## Example
+
+The following C# code shows how to parse an xml string:
+
+```csharp
+XElement contacts = XElement.Parse(
+    @"<Contacts>
+        <Contact>
+            <Name>Patrick Hines</Name>
+            <Phone Type=""home"">206-555-0144</Phone>
+            <Phone Type=""work"">425-555-0145</Phone>
+            <Address>
+            <Street1>123 Main St</Street1>
+            <City>Mercer Island</City>
+            <State>WA</State>
+            <Postal>68042</Postal>
+            </Address>
+            <NetWorth>10</NetWorth>
+        </Contact>
+        <Contact>
+            <Name>Gretchen Rivas</Name>
+            <Phone Type=""mobile"">206-555-0163</Phone>
+            <Address>
+            <Street1>123 Main St</Street1>
+            <City>Mercer Island</City>
+            <State>WA</State>
+            <Postal>68042</Postal>
+            </Address>
+            <NetWorth>11</NetWorth>
+        </Contact>
+    </Contacts>");
+Console.WriteLine(contacts);
+```
+
+The root `Contacts` node has two `Contact` nodes. To access some specific data in your parsed xml, use the `Elements` method. The following example prints the first `Contact` node to the console:
```

Comment:
```suggestion
The root `Contacts` node has two `Contact` nodes. To access some specific data in your parsed XML, use the <xref:System.Xml.Linq.XElement.Elements?displayProperty=nameWithType> method, which in this case returns the child elements of the root `Contacts` node. The following example prints the first `Contact` node to the console:
```

Reply:
@rpetrusha, There is a problem if the xref is used here.
<https://docs.microsoft.com/en-us/dotnet/api/system.xml.linq.xelement.elements> (404 Not found). I've opened https://github.com/dotnet/dotnet-api-docs/issues/3260 when I was working on this.

Reply:
You can change that to use [XElement.Elements()](xref:System.Xml.Linq.XContainer.Elements) since the API is defined by the parent class.

Reply:
Thanks, @mairaw.

Reply:
```suggestion
The root `Contacts` node has two `Contact` nodes. To access some specific data in your parsed XML, use the [XElement.Elements()](xref:System.Xml.Linq.XContainer.Elements) method, which in this case returns the child elements of the root `Contacts` node. The following example prints the first `Contact` node to the console:
```